src/Symfony/Component/Serializer/Normalizer/ObjectNormalizer.php

```diff
@@ -98,7 +98,10 @@ protected function extractAttributes(object $object, ?string $format = null, arr
             $name = $reflMethod->name;
             $attributeName = null;
 
-            if (str_starts_with($name, 'get') || str_starts_with($name, 'has') || str_starts_with($name, 'can')) {
+            if (
+                (str_starts_with($name, 'get') || str_starts_with($name, 'has') || str_starts_with($name, 'can'))
+                && ctype_upper($name[3] ?? '')
+            ) {
```

Comment:
```suggestion
            if ((str_starts_with($name, 'get') || str_starts_with($name, 'has') || str_starts_with($name, 'can')) && ctype_upper($name[3] ?? '')) {
```

Lets use one line please